hail/python/hail/genetics/pedigree.py

```diff
@@ -161,27 +169,18 @@ class Pedigree(object):
     """
 
     def __init__(self, trios):
-        self._jrep = Env.hail().methods.Pedigree(jindexed_seq([t._jrep for t in trios]))
-        self._trios = trios
-
-    @classmethod
-    def _from_java(cls, jrep):
-        ped = Pedigree.__new__(cls)
-        ped._jrep = jrep
-        ped._trios = None
-        super(Pedigree, ped).__init__()
-        return ped
+        self._trios = tuple(trios)
```

Comment:
Suggestion: Add a type checker for this or an assertion that trios is a list of Trio objects